### PR TITLE
Use pytest instead of nose

### PR DESCRIPTION
**Pull request by arokem**

### CODE REVIEW

**Comment by arokem** [comment]
Supersedes #40

File: shablona/due.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+"""
+
+Stub file for a guaranteed safe import of duecredit constructs:  if duecredit
+is not available.
+
+To use it, place it into your project codebase to be imported, e.g. copy as
+
+    cp stub.py /path/tomodule/module/due.py
+
+Note that it might be better to avoid naming it duecredit.py to avoid shadowing
```
Comment (QuLogic) [inline shablona/due.py:13]: That's not possible when you enable absolute imports (#41).

Comment (arokem) [inline shablona/due.py:13]: What is not possible? Shadowing? Are you suggesting we change the wording here? To what?

Comment (arokem) [inline shablona/due.py:13]: FWIW, that code is already on `master`: https://github.com/uwescience/shablona/blob/master/shablona/due.py, so if you have suggestions a PR is probably the best way to go.

Comment (QuLogic) [inline shablona/due.py:13]: Yes, the shadowing would not be possible with absolute imports enabled.

**Pull request merged**